fearless/Common/Configs/ApplicationConfigs.swift

```diff
@@ -103,7 +103,7 @@ extension ApplicationConfig: ApplicationConfigProtocol {
     }
 
     var chainListURL: URL {
-        URL(string: "https://raw.githubusercontent.com/soramitsu/fearless-utils/master/chains/chains.json")!
+        URL(string: "https://raw.githubusercontent.com/soramitsu/fearless-utils/master/chains/chains_dev.json")!
```

Comment:
I think it worth to leave a TODO here or even include such strings to build settings to avoid potential issues with switching to production URLs